packages/authentication/src/types.ts

```diff
@@ -51,3 +51,54 @@ export interface AuthenticationStrategy {
    */
   authenticate(request: Request): Promise<UserProfile | undefined>;
 }
+
+export interface UserService<U, C> {
```

Comment:
We created the interface for services in https://github.com/strongloop/loopback-next/blob/master/packages/authentication/src/services/user.service.ts
and 
https://github.com/strongloop/loopback-next/blob/master/packages/authentication/src/services/token.service.ts
No need to duplicate the definition here :-p